humble/distribution.yaml

```diff
@@ -10737,6 +10737,22 @@ repositories:
       url: https://github.com/ros-visualization/tango_icons_vendor.git
       version: humble
     status: maintained
+  tecgihan_driver:
+    doc:
+      type: git
+      url: https://github.com/tecgihan/tecgihan_driver.git
+      version: main
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/tecgihan/tecgihan_driver-release.git
```

Comment:
Do you intend to release your package into the ROS 2 Rolling distribution? If so, we [recommend using a release repo under github.com/ros2-gbp](https://docs.ros.org/en/rolling/How-To-Guides/Releasing/Release-Team-Repository.html#what-if-my-existing-release-repo-isn-t-on-ros2-gbp) (you can [request that a repository be created for you under github.com/ros2-gbp](https://docs.ros.org/en/rolling/How-To-Guides/Releasing/Release-Team-Repository.html#create-a-new-release-repository)). If not, then we can proceed with https://github.com/tecgihan/tecgihan_driver-release.git.

Reply:
Thank you for your review.
We currently have no plans to release this package into the ROS 2 Rolling distribution.
Accordingly, we would appreciate it if you could proceed with https://github.com/tecgihan/tecgihan_driver-release.git.

Reply:
Thank you for the information!